src/test/java/api/support/fixtures/CheckOutFixture.java

```diff
@@ -0,0 +1,124 @@
+package api.support.fixtures;
+
+import static api.support.APITestContext.getOkapiHeadersFromContext;
+import static api.support.http.InterfaceUrls.checkOutByBarcodeUrl;
+import static api.support.http.InterfaceUrls.overrideCheckOutByBarcodeUrl;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+
+import api.support.RestAssuredClient;
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import io.vertx.core.json.JsonObject;
+
+import api.support.builders.OverrideCheckOutByBarcodeRequestBuilder;
+
+public class CheckOutFixture {
```

Comment:
Given this class is new, please could the parameter formatting be fixed (by using standard line continuation and indentation) in this initial version.

Apologies, it's likely my poor formatting that started this structure :-/

Reply:
Please could the same be done for the other new fixtures as well.